lib/versioning/ruby/index.spec.ts

```diff
@@ -505,6 +505,8 @@ describe('semverRuby', () => {
 
     it('returns correct version for replace strategy', () => {
       [
+        ['< 1.2.5', '< 1.0.3', 'auto', '1.0.3', '1.2.4'],
+        ['< 1.2.5', '< 1.0.3', 'widen', '1.0.3', '1.2.4'],
```

Comment:
What's this?

Reply:
Actually, it's a good question. I don't know how other changes broke the coverage, but these lines fix it. Need to trace this down, it can be some deep bug here.

Reply:
Best to figure it out :)